go/vt/sqlparser/ast_rewriting_test.go

```diff
@@ -273,22 +270,22 @@ func TestRewrites(in *testing.T) {
 		expected: "select * from tbl where id regexp '%foobar'",
 	}, {
 		in:       "SELECT * FROM tbl WHERE exists(select col1, col2 from other_table where foo > bar)",
-		expected: "SELECT * FROM tbl WHERE exists(select 1 from other_table where foo > bar limit 1)",
```

Comment:
The `limit 1` logic can move to the operator model. For now we've lost this optimisation, but it should be easy to restore in a follow-up PR. It's not hugely important - if we push this down to MySQL, it does it on it's own. It's mostly useful when we execute the semi join on the vtgate side